### PR TITLE
Updated footer to include LGL

### PR DESCRIPTION
**Pull request by seanherron**

### CODE REVIEW

File: peacecorps/peacecorps/templates/donations/base.jinja [hunk 1]
```diff
@@ -164,41 +167,52 @@
...
+        <p class="t-body--md">All links return to peacecorps.gov.</p>
         <hr class="separator--h" />
         <nav class="column nested_column--fourth sub_nav sub_nav--vertical">
-          <h3 class="t-title--3 t--gray--2">Join</h3>
+          <h3 class="t-title--3 t--gray--2">Get Involved</h3>
           <ul class="u-unmarked_list">
             <li class="sub_nav__link">
-              <a href="http://www.peacecorps.gov/apply/?from=applyby">Apply</a>
+              <a href="http://donate.peacecorps.gov">
+                Donate</a>
```
Comment (cmc333333) [inline peacecorps/peacecorps/templates/donations/base.jinja:177]: This is a link that does not return to peacecorps.gov. Also, shouldn't it be beta.peacecorps.gov/donate/ ?

File: peacecorps/peacecorps/templates/donations/base.jinja [hunk 2]
```diff
@@ -164,41 +167,52 @@
...
-              <a href="http://www.peacecorps.gov/volunteer/learn/meet/events/">
-                Attend an event</a>
+              <a href="http://www.peacecorps.gov/resources/returned/home/">
+                Returned Volunteers</a>
             </li>
             <li class="sub_nav__link">
-              <a href="http://www.peacecorps.gov/openings/?from=hpb">
-                Current opportunities</a>
+              <a href="http://www.peacecorps.gov/resources/faf/">
+                Family & Friends</a>
```
Comment (cmc333333) [inline peacecorps/peacecorps/templates/donations/base.jinja]: `&amp;` pretty please

Comment (seanherron) [inline peacecorps/peacecorps/templates/donations/base.jinja]: Fixed

File: peacecorps/peacecorps/templates/donations/base.jinja (hunk 1 above)
Comment (seanherron) [inline peacecorps/peacecorps/templates/donations/base.jinja:177]: Good point - I'm trying to make this consistent across what we have for LGL, and I'm not sure of a better way to do it. Ideas?

Comment (cmc333333) [inline peacecorps/peacecorps/templates/donations/base.jinja:177]: Got nothing. Can merge though!

**Pull request merged**